packages/frontend/src/components/SimpleStatistic/index.tsx

```diff
@@ -10,19 +11,29 @@ import {
 } from './SimpleStatistics.styles';
 
 const SimpleStatistic: FC = () => {
-    const { resultsData, bigNumber, bigNumberLabel, isLoading } =
-        useVisualizationContext();
+    const {
+        resultsData,
+        isLoading,
+        bigNumberConfig: { bigNumber, bigNumberLabel, bigNumberStyle, field },
+    } = useVisualizationContext();
 
     const validData = bigNumber && resultsData?.rows.length && bigNumberLabel;
 
     if (isLoading) return <LoadingChart />;
 
+    const formattedBigNumber = formatValue(
+        field?.format,
+        field?.round,
+        bigNumber,
+        bigNumberStyle,
+    );
```

Comment:
I expected this to be done inside useBigNumberConfig so all the business logic is closer.

Reply:
I suggested that on another thread, but I thought you would say that there should not be visualization logic inside the `config` or similar :) I don't mind moving the logic there if you think that is the right place.